src/containers/AudioUploader/components/AudioManuscript.tsx

```diff
@@ -160,7 +160,7 @@ const AudioManuscript = ({ audio, audioLanguage = "no" }: AudioManuscriptProps)
       setStatus({ status: "manuscript" });
     } else if (polledData?.status === "FAILED" && isPolling) {
       setIsPolling(false);
-      toast.error({ title: "textGeneration.failed.transcription" });
+      toast.error({ title: t("textGeneration.failed.transcription") });
```

Comment:
Fint at du har fikset denne, men mistenker at samme tilfellet gjelder for brorparten av bruket her. Ser ut som at du har droppet `t` på hvertfall en til i denne filen. Usikker på andre filer.